.kokoro/nightly/downstream-protobuf-source-compatibility.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# Comma-delimited list of repos to test with the local java-shared-dependencies
+if [ -z "${REPOS_UNDER_TEST}" ]; then
+  echo "REPOS_UNDER_TEST must be set to run downstream-protobuf-source-compatibility.sh"
```

Comment:
Can we provide more detailed error hint if they are not supplied? For example, what are the expected format of the input etc. Same comment for `PROTOBUF_RUNTIME_VERSION` below.

Reply:
Sure, I can add an example